cmd2.py

```diff
@@ -1007,7 +1026,18 @@ def onecmd(self, line):
             func = getattr(self, funcname)
         except AttributeError:
             return self.default(statement)
-        stop = func(statement)
+
+        if self.use_argument_list:
```

Comment:
We can't do this in **onecmd**.   This breaks any underlying calls which are actually implemented by Python's built-in ``cmd`` module which ``cmd2`` extends - most importantly **help**.  With this in place doing a "help history" will  throw a "TypeError" exception because the underlying ``cmd`` help commands expects a string.

Additionally, if someone tries to upgrade their ``cmd`` app to use ``cmd2``, at first none of their commands would work.